mesh/mesh.cpp

```diff
@@ -1552,6 +1552,153 @@ Geometry::Type Mesh::GetTypicalElementGeometry() const
    return geom;
 }
 
+void Mesh::GetExteriorFaceMarker(Array<int> & face_marker) const
+{
+   if (face_marker.Size() < faces_info.Size())
+   {
+      face_marker.SetSize(faces_info.Size());
+   }
+
+   for (int f = 0; f < faces_info.Size(); f++)
+   {
+      if (faces_info[f].NCFace == -1 &&
+          faces_info[f].Elem2No < 0 &&
+          faces_info[f].Elem2Inf < 0 )
+      {
+         face_marker[f] = 1;
+      }
+      else
+      {
+         face_marker[f] = 0;
+      }
+   }
+}
+
+void Mesh::RemoveInternalBoundaries(Array<int> &bdr_marker, bool excl) const
```

Comment:
How about renaming these from `Remove` to `Unmark`? It is more symmetric with the `Mark` functions and also `Remove` a little bit provokes the idea it might modify the mesh 🤔 .